modules/openapi/api/apis/scheduler/scheduler_nodelabel_list.go

```diff
@@ -22,7 +22,7 @@ import (
 var SCHEDULER_NODELABEL_LIST = apis.ApiSpec{
```

Comment:
move to apis/orchestrator better, and rename variable to OCHESTRATOR_NODELABEL_LIST

Reply:
moved those files